pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -141,6 +142,18 @@ public SuccessResponse addTable(String tableConfigStr) {
     }
   }
 
+  @PUT
+  @Produces(MediaType.APPLICATION_JSON)
```

Comment:
We should also use @Consumes annotation right?

Reply:
got it

Reply:
I stayed with only `@Produces(MediaType.APPLICATION_JSON)` to follow the pattern of other APIs